AutomatedNotebooks-Manager.ipynb

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
<p>Wonder whether it might be better to create a wrapped function - log_item(log_str) - that opens and closes the file each time it writes.</p><p>Or even better use the Python logger - I've been thinking of adding this to Msticpy for a while. https://docs.python.org/3/howto/logging.html#logging-basic-tutorial</p><p>Wish I'd had that thought 2 years ago.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>